charts/applicationset-progressive-sync/Chart.yaml

```
@@ -10,13 +10,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.0-prealpha
+version: 0.7.1-prealpha
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "sha-f70b92b"
+appVersion: "v0.7.1-prealpha"
```

```suggestion
appVersion: "v0.7.0-prealpha"
```
Sorry, I didn't pay enough attention in the previous suggestion :)